portfolio/src/main/webapp/script.js

```diff
@@ -65,9 +65,9 @@ function addTextToDom(text) {
     sessionStorage["size"] = parsedText.numComments;
 
     var i;
-    for (i = 0; i < parsedText.comments.length; i++) {
-        const comment = parsedText.comments[i];
-        const heading = ` (${comment.email}, posted at ${comment.date})`;
+    for (i = 0; i < parsedText.length; i++) {
+        const comment = parsedText[i];
+        const heading = ` (posted at ${comment.date})`;
```

Comment:
Don't forget what I mentioned about switching from UTC to local time and vice versa when dealing with dates and times.

Reply:
done!